openmetadata-ui/src/main/resources/ui/src/pages/DatabaseSchemaPage/SchemaTablesTab.tsx

```diff
@@ -11,20 +11,19 @@
  *  limitations under the License.
  */
 
-import { Col, Row, Switch, Table as TableAntd, Typography } from 'antd';
+import { Col, Row, Switch, Typography } from 'antd';
 import { ColumnsType } from 'antd/lib/table';
 import DescriptionV1 from 'components/common/description/DescriptionV1';
 import ErrorPlaceHolder from 'components/common/error-with-placeholder/ErrorPlaceHolder';
 import NextPrevious from 'components/common/next-previous/NextPrevious';
 import RichTextEditorPreviewer from 'components/common/rich-text-editor/RichTextEditorPreviewer';
-import Loader from 'components/Loader/Loader';
+import TableAntd from 'components/common/Table/Table';
```

Comment:
```suggestion
import Table from 'components/common/Table/Table';
```

Reply:
we have a clashing name with Type Table.